docs/docs/releases.md

```diff
@@ -163,13 +163,35 @@ echo "deb https://dl.cloudsmith.io/public/pomerium/pomerium/deb/debian buster ma
 brew tap pomerium/tap
 brew install pomerium-cli
 ```
+### Docker Image
+
+The CLI utilizes a [minimal](https://github.com/GoogleContainerTools/distroless) [docker container](https://www.docker.com/resources/what-container). You can find the Pomerium CLI image on [dockerhub](https://hub.docker.com/r/pomerium/cli). It can be pulled in several flavors and architectures.
```

Comment:
Consider adding an overview to Docker Hub before linking to it.

Reply:
That will come from the README in that repo when populated.